rag/nlp/term_weight.py

```
@@ -89,10 +118,12 @@ def load_dict(fnm):
                 self.ne = json.load(f)
         except Exception:
             logging.warning("Load ner.json FAIL!")
-        try:
-            self.df = load_dict(os.path.join(fnm, "term.freq"))
-        except Exception:
-            logging.warning("Load term.freq FAIL!")
+        freq_path = os.path.join(fnm, "term.freq")
+        if os.path.exists(freq_path):
+            try:
+                self.df = load_dict(freq_path)
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)